statsmodels/stats/_lilliefors.py

```diff
@@ -87,55 +107,131 @@ def ksstat(x, cdf, alternative='two_sided', args=()):
-                           [187, 196, 208, 226, 262, 305],
-                           [181, 190, 201, 219, 254, 296],
-                           [176, 184, 195, 213, 247, 287],
-                           [171, 179, 190, 207, 240, 279],
-                           [167, 175, 185, 202, 234, 273],
-                           [163, 170, 181, 197, 228, 266],
-                           [159, 166, 176, 192, 223, 260],
-                           [143, 150, 159, 173, 201, 236],
-                           [131, 138, 146, 159, 185, 217],
-                           [115, 120, 128, 139, 162, 189],
-                           [ 74,  77,  82,  89, 104, 122],
-                           [ 37,  39,  41,  45,  52,  61],
-                           [ 25,  26,  28,  30,  35,  42]])[:,::-1] / 1000.
-
-
-    lf = TableDist(alpha, size, crit_lf)
+        #critical values, rows are by sample size, columns are by alpha
+        crit_lf = np.array(   [[303, 321, 346, 376, 413, 433],
+                               [289, 303, 319, 343, 397, 439],
+                               [269, 281, 297, 323, 371, 424],
+                               [252, 264, 280, 304, 351, 402],
+                               [239, 250, 265, 288, 333, 384],
+                               [227, 238, 252, 274, 317, 365],
+                               [217, 228, 241, 262, 304, 352],
+                               [208, 218, 231, 251, 291, 338],
+                               [200, 210, 222, 242, 281, 325],
+                               [193, 202, 215, 234, 271, 314],
+                               [187, 196, 208, 226, 262, 305],
+                               [181, 190, 201, 219, 254, 296],
+                               [176, 184, 195, 213, 247, 287],
+                               [171, 179, 190, 207, 240, 279],
+                               [167, 175, 185, 202, 234, 273],
+                               [163, 170, 181, 197, 228, 266],
+                               [159, 166, 176, 192, 223, 260],
+                               [143, 150, 159, 173, 201, 236],
+                               [131, 138, 146, 159, 185, 217],
+                               [115, 120, 128, 139, 162, 189],
+                               [ 74,  77,  82,  89, 104, 122],
+                               [ 37,  39,  41,  45,  52,  61],
+                               [ 25,  26,  28,  30,  35,  42]])[:,::-1] / 1000.
+
+
+        # also build a table for larger sample sizes
+        def f(n):
+            return np.array([.736/np.sqrt(n), .768/np.sqrt(n),
+            .805/np.sqrt(n), .886/np.sqrt(n), 1.031/np.sqrt(n)])
+
+        higher_sizes = np.array([35, 40, 45, 50, 60, 70,
+                                80, 100, 200, 500, 1000,
+                                2000, 3000, 5000, 10000, 100000], float)
+        higher_crit_lf = np.zeros([higher_sizes.shape[0], crit_lf.shape[1]-1])
+        for i in range(len(higher_sizes)):
+            higher_crit_lf[i,:] = f(higher_sizes[i])
```

Comment:
flake8 `[i,:]` --> `[i, :]`.  Easier to catch these here than try to fix them up later.